### DISCUSSION TITLE
The First Remote Meetup About Web3

### DISCUSSION THREAD

**Discussion by samnang** [web3together/web3together#9] [Q&A] [answered]
I'm so excited in having the first meetup with our members. It's a good opportunity for us to know each other more, and we hope we will get some more feedbacks what interests them in joining to learn about Web3.

In the first meetup, we will cover some topics:

1. Why Blockchain?
2. Why Web3?
3. What are common use cases for decentralized applications?

First thing first, I would like to survey our members' availability :

* ~~👍 Thursday 25th, Nov 2021 at 6-7pm UTC+7~~ 
* ❤️ [Saturday 27th, Nov 2021 at 10-11am UTC+7](https://teams.live.com/meet/95539752700705)

Please let us know your availability to join with the reaction emoji above.

**Comment by borinmin** [comment]
I could join on Saturday, bong.

**Reply by samnang** [reply]
Could you help to respond by reacting with an emoji above:

![image](https://user-images.githubusercontent.com/131172/142769614-6f7332d2-412d-490e-9cf4-48f8e9d4ab5e.png)

**Comment by samnang** [comment, marked as answer]
Okay, let's go our first event on Saturday 27th, Nov 2021 at 10-11am. 

Join the meeting:
https://teams.live.com/meet/95539752700705

**Reply by CHOMNANP** [reply]
Hi bong @samnang, thanks for arranging the meeting.
It has been a great time having a discussion with everyone.
I'm quite curious on how we could join the new telegram group.

**Reply by samnang** [reply]
@CHOMNANP you can join the telegram group here: https://t.me/TheWeb3Together
